src/components/Header.js

```diff
@@ -19,11 +19,18 @@ class Header extends Component {
           {isAuthenticated && (
             <div className="right menu">
               {isAdmin && (
-                <div className="right menu">
-                  <Link to="/admins" className="item">
-                    <i className="address book outline icon" /> Admins
-                  </Link>
-                </div>
+                <>
```

Comment:
is it a react fragment?

Reply:
Ele mesmo! tanto faz fazer `<React.Fragment>` ou `<>`